pycbc/workflow/coincidence.py

```diff
@@ -546,6 +546,68 @@ def setup_interval_coinc(workflow, hdfbank, trig_files, stat_files,
     logging.info('...leaving coincidence ')
     return statmap_files
 
+def setup_multiifo_interval_coinc_inj(workflow, hdfbank, full_data_trig_files, inj_trig_files,
+                                      stat_files, veto_file, veto_name,
+                                      out_dir, pivot_ifo, fixed_ifo, tags=None):
+    """
+    This function sets up exact match multiifo coincidence for injections
+    """
+    if tags is None:
+        tags = []
+    make_analysis_dir(out_dir)
+    logging.info('Setting up coincidence')
+
+    if len(hdfbank) != 1:
+        raise ValueError('Must use exactly 1 bank file for this coincidence '
+                         'method, I got %i !' % len(hdfbank))
+    hdfbank = hdfbank[0]
+    # Wall time knob and memory knob
+    factor = int(workflow.cp.get_opt_tags('workflow-coincidence', 'parallelization-factor', tags))
+
+    ffiles = {}
+    ifiles = {}
+    ifos, files = full_data_trig_files.categorize_by_attr('ifo')
+    for ifo, file in zip(ifos, files):
+        ffiles[ifo] = file[0]
+    ifos, files = inj_trig_files.categorize_by_attr('ifo')
+    for ifo, file in zip(ifos, files):
+        ifiles[ifo] = file[0]
+
+    injinjFileList = FileList()
+    injfullFileList = FileList()
+    fullinjFileList = FileList()
+    for ifo in ifos:
+        if ifo == pivot_ifo:
```

Comment:
perhaps add a comment here describing the logic, ie separating 'pivot ifo' on one side from all the others